packages/renderer/src/components/AuthProvider/component.tsx

```diff
@@ -95,8 +96,10 @@ export const AuthProvider: React.FC<{ children: ReactNode }> = ({ children }) =>
 
   useEffect(() => {
     if (wallet && chainId) {
-      dispatch(fetchENSList({ address: wallet, chainId }));
+      dispatch(fetchENSList({ address: wallet, chainId: ChainId.ETHEREUM_SEPOLIA }));
       dispatch(identify({ userId: wallet }));
+      dispatch(fetchTiles({ chainId: ChainId.ETHEREUM_SEPOLIA }));
+      dispatch(fetchLandList({ address: wallet, chainId: ChainId.ETHEREUM_SEPOLIA }));
```

Comment:
Are these sepolia chain ids meant to be here or were there for testing?

Reply:
good catch, removed! were there for testing